Iterable/iterable-web-sdk
# Add Users UI to React App

## JIRA Ticket(s) if any

* [MOB-3930](https://iterable.atlassian.net/browse/MOB-3930)

## Description

Adds /users view to hit the /users namespace endpoints from the React UI

Cypress tests coming in the next PR

## Test Steps

1. Run a local JWT generation backend by [cloning this repo](https://github.com/Iterable/jwt-generator) and running `docker-compose up -d`
2. Run react example app in this repo with `yarn install:all && yarn start:all:react`
3. Open up `localhost:8080` in your browser and use the login form in the header to login with an email
   * I suggest using a throwaway email
5. Navigate to users page
6. Fill out each form and ensure the endpoint is hit as advertised with the data you typed in
